tests/page-objects/apdex/load.page.js

```diff
@@ -0,0 +1,36 @@
+const Page = require('@page-objects/apdex/page');
+const CUSTOM_INSTANCE = 'Custom';
+
+class LoadPage extends Page {  
+  
+  get inputInstanceUrl() {
+    return $('//android.widget.EditText[@resource-id="org.medicmobile.webapp.mobile:id/txtAppUrl"]');
+  }
+
+  get btnSave() {
+    return $('//android.widget.Button[@resource-id="org.medicmobile.webapp.mobile:id/btnSaveSettings"]');
+  }
+
+  async loadInstance(settingsProvider) {
```

Comment:
settingsProvider is still passed as a parameter here. is this a miss?

Reply:
Nope - same as above `await this.inputInstanceUrl.setValue(super.settingsProvider.getInstanceURL());` returns undefined

Reply:
ok i got this... thanks